plugins/polio/api.py

```diff
@@ -1095,8 +1096,16 @@ def handle_ona_request_with_key(request, key, country_id=None):
         if country_id is not None and config.get("country_id", None) != cid:
             continue
         forms = get_url_content(
-            url=config["url"], login=config["login"], password=config["password"], minutes=config.get("minutes", 60)
+            url=config["url"] + "?date_created__month__lte=24",
+            login=config["login"],
+            password=config["password"],
+            minutes=config.get("minutes", 60),
         )
+        if not forms:
```

Comment:
I would put the try except at this level so it doesn't affect the other place we could use get_url_content  and more info about the error could be included in the e-mail